### ISSUE TITLE
[Question] Can i use a SELECT query for template variables with InfluxDB 0.11?

### ISSUE THREAD

**Issue by sofixa** [grafana/grafana#4968]
Hello, I am running 
```
Version 3.0.0-beta5 (commit: v3.0-beta4+14-gd98839f)
```
on Debian Jessie. 

Due to a [bug in InfluxDB](https://github.com/influxdata/influxdb/issues/6581) i cannot use a SHOW TAG VALUES command to get the template variable i need, so i tried using a SELECT query, like this:
```
select datastore,cpu_corecount_provisioned_average from virtualmachine WHERE "name" =~ /$name/
```
The query returns the following in InfluxDB:
```
> select datastore,cpu_corecount_provisioned_average from virtualmachine where host =~ /vcenter-01/ limit 10
name: virtualmachine
--------------------
time                    datastore                       cpu_corecount_provisioned_average
1462788369000000000     netapp03a_ds06           1
1462788369000000000     netapp03b_ds04             1
1462788369000000000     netapp02b_ds05                  4
1462788369000000000     netapp02b_ds03                  2
1462788369000000000     netapp03b_ds02       6
1462788369000000000     netapp02a_ds02                  6
1462788369000000000     netapp03b_ds06           4
1462788369000000000     netapp03a_ds03             4
1462788369000000000     netapp03b_ds01       2
1462788369000000000     netapp02b_ds01                  12
```

_The_ value i actually need is datastore, i've included cpu_corecount_provisioned_average  because InfluxDB doesn't permit a SELECT on a tag_

But this doesn't work, because Grafana takes the first column with results, which is the timestamp.
Is there a way to go around this? I haven't found a way to format InfluxDB's response so that i don't see the timestamp, nor to force Grafana to use another column.

Thanks in advance

**Comment by torkelo** [comment]
Select style queries are not supported when used from templating. So if you cant use a SHOW query there is no solution (currently)

**Comment by BRNTZN** [comment]
Not sure why this should only be possible for tags. I'd like to see a fix for this.

**Comment by hanynowsky** [comment]
The SHOW tags values is so poor in InfluxDB. Using select queries in grafana templating is a blocker feature.

**Comment by marefr** [comment]
Should be fixed by #5013 and included in Grafana v5.2.0

**Comment by simplyviks** [comment]
can somebody please confirm if this was merged to Grafana v5.2.0?
thanks in advance!!
